runtime/jcl/common/sun_reflect_ConstantPool.c

```diff
@@ -47,12 +47,14 @@ clearException(J9VMThread *vmThread)
 static SunReflectCPResult
 getRAMConstantRefAndType(J9VMThread *vmThread, jobject constantPoolOop, jint cpIndex, UDATA *cpType, J9RAMConstantRef **ramConstantRef)
 {
-	J9Class *ramClass = J9CLASS_FROMCPINTERNALRAMCLASS(vmThread, constantPoolOop);
+	printf("50, %p\n", constantPoolOop);
+	J9ConstantPool *ramConstantPool = J9CONSTANTPOOL_FROMCPINTERNALCONSTANTPOOL(vmThread, constantPoolOop);
+	printf("52, %p\n", ramConstantPool);
```

Comment:
to test you can add an assert to make sure that on compressefs the most significant 32bits are all zeroes